dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -10,6 +12,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 import java.util.Properties;
+import java.util.Map;
+import java.util.HashMap;
```

Comment:
No longer required after the move to using a `TypeTransformer` object. I'll remove these.